src/main/java/com/bitwig/extensions/controllers/akai/mpk_mini_mk3/MpkMiniMk3ControllerExtension.java

```diff
@@ -1,227 +0,0 @@
-package com.bitwig.extensions.controllers.akai.mpk_mini_mk3;
```

Comment:
Why did you remove this controller?

Reply:
That was also a mistake. I confused it with APC. It should not be removed.

Reply:
Please double check your changes and let me know when it's good :+1:

Reply:
I'll see if I can get it done around noon.